cli/types/cypress.d.ts

```diff
@@ -1884,6 +1884,25 @@ declare namespace Cypress {
      */
     spread(fn: (...args: any[]) => void): Chainable<Subject>
 
+    /**
+     * Enables running Cypress commands in a secondary domain
```

Comment:
This may just be semantics, but would it make more sense to say 'non-primary domain' instead of 'secondary domain' since it could be a domain past the secondary.

Reply:
Primary/non-primary is more unambiguously binary, but I like having the two terms to be more distinct (one not including the other, which may be confusing in a conversation). We mostly use "secondary" in comments now, but I imagine the terminology will be used in the documentation, so we should reach a consensus on it before that's written. Let's plan to discuss it with DX.